modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/fieldSupport.js

```diff
@@ -353,25 +350,25 @@ export const normalizeSettingsContextPages = (
 				});
 			}
 
-			const newInstanceId = generateInstanceId(8);
+			const instanceId = generateInstanceId(8);
```

Comment:
I think this change may conflict with the changes I made in this [PR](https://github.com/brianchandotcom/liferay-portal/pull/107126/files), so please rebase it before  try to run CI tests again.

The rebased code  will look like this:
```suggestion
			const instanceId = generateInstanceId();
```

Reply:
I did rebase with bchan/master and resolved the conflict.